src/Stepper/Stepper.jsx

```diff
@@ -0,0 +1,129 @@
+import React, {PropTypes} from 'react';
+
+import Mixins from '../mixins';
+import {getMuiTheme} from '../styles';
+
+const Stepper = React.createClass({
+  propTypes: {
+    /**
+     * The current active step index which passed by parent component
+     */
+    activeStepIndex: PropTypes.number,
+    /**
+     * Children should be Step type
+     */
+    children: PropTypes.node,
+    /**
+     * The icon will show when the step is completed
+     */
+    completedIcon: PropTypes.node,
```

Comment:
What if this prop was renamed to `icon` and the PropType was a `node` or a `function`? If you pass a function, it will be called with the props of the `<Step />` and the developer could return the appropriate icon based on props (for example a different icon based on `isComplete`, `isHovered`, `isActive`, or any combination of those, etc.)

Don't implement that idea yet, just want to throw the idea out there.